src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -228,4 +255,9 @@ private static function buildPathsCompletion(array $paths, string $prefix = ''):
 
         return $completionPaths;
     }
+
+    private function getAvailableFormatOptions(): array
+    {
+        return ['yaml', 'json'];
```

Comment:
nitpick:
consider using enums, symfony ~6.1 targets php >=8.1

Reply:
also this method might be the nice place to check for the availability of `yaml` formatter

Reply:
Using an enum does not make sense. The option completion need an array of strings, and the value of the option will be a string. Enums won't bring any benefit here.